### DISCUSSION TITLE
Empty Dropdown Using ModelSelect2Widget

### DISCUSSION THREAD

**Discussion by celine-setyawan** [codingjoe/django-select2#45] [Q&A]
Hello, thank you for the opensource, this is my first time using django-select2.

I don't know why my dropdown won't show anything. This is the frontend. I want those 4 fields chained. But they dont even show any data. 
![image](https://user-images.githubusercontent.com/54545889/87085337-e77f9f00-c259-11ea-9edd-382779abdc0e.png)'

Seems like I have similar problem like [this issue](https://github.com/applegrew/django-select2/issues/568). I've tried the solution but it doesn't work for me. If I remove the widgets=ModelSelect2Widget, my dropdown is populated with data from the DB, so I guess the problem lies with the ModelSelect2Widget. I use django-select2==7.4.2 and django==3.0.7

This is my **forms.py**
```python
class FormSekolah(ModelForm):
          provinsi = ModelChoiceField(
                          queryset=Provinsi.objects.all(),
                          widget=ModelSelect2Widget(
                                              attrs={
                                                       'class': 'form-control form-control-sm',
                                                       'data-placeholder': '-- Pilih Provinsi --',
                                                       'data-minimum-input-length': 0
                                                },
                           model=Provinsi,
                           search_fields=['nama__icontains'],
                           dependent_fields={'kokab': 'cities'}
                           )
         )

        class Meta:
                  model = Sekolah
                  fields = [
                              # other field,
                              'provinsi',
                   ]
```

This is my **models.py**
```python
class Provinsi(models.Model):
    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
    nama = models.CharField(max_length=50)
    negara = models.ForeignKey(Negara, on_delete=models.CASCADE, related_name="provinces")

    def __str__(self):
        return self.nama
```

Do I need to do something with my **views.py** (beside create and save forms and render to the html) or even create my own **.js** ?

Thanks before.

**Comment by tow3r91** [comment]
I have the same problem. Some answer???

**Comment by codingjoe** [comment]
Sorry, I somehow must have missed a notification here. @celine-setyawan do you still have this problem? I see that you use `dependent_fields`. Can you maybe also provide the fields specified in there?

@tow3r91 if you have a similar problem, maybe you could provide some examples too?
